force-app/main/default/classes/OrgConfig.cls

```diff
@@ -61,6 +61,11 @@ public inherited sharing class OrgConfig {
         return validCountriesByCode;
     }
 
+    //todo: add test coverage
+    public Boolean isPersonAccountsEnabled() {
```

Comment:
If this method is going to be added here then I think both the selector and the "contact merge" class should both reference this same method.